openedx/tutor-contrib-superset
# feat: enable DASHBOARD_RBAC

## Description

This PR enables DASHBOARD_RBAC and modifies the default role to be compliant with our implementation.

## Steps to test:

Assuming you have more than one course at least 1 event from those courses.

1. Register a new user
2. Grant course staff permissions to the user for just one course.
3. Open superset with this user.
4. Create a dashboard and a chart that queries the `xapi_events_all_parsed` dataset.
5. Assign the role `Open edX` to the chart in the properties.
6. Compare the records the created user has vs the superuser.
7. See the difference.

In the right is the staff user, in the left is the superuser.

![image](https://github.com/openedx/tutor-contrib-superset/assets/33465240/c47bb5c5-e559-4a43-8729-d96f400726b0)

![image](https://github.com/openedx/tutor-contrib-superset/assets/33465240/6b5bf8fe-bad6-4278-9540-917f0e9c26ca)


This confirms that the current implementation of permissions is compatible with the RBAC feature.

This PR fixes:
- https://github.com/openedx/tutor-contrib-oars/issues/27
- https://github.com/openedx/tutor-contrib-oars/issues/15
